parser/parser.go

```diff
@@ -2,12 +2,14 @@ package parser
 
 import (
 	"bufio"
-	"bytes"
```

Comment:
we still the bytes package in other parts of this file

Reply:
Fixed!